src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 5;
+    private final Random random = new Random();
+
+    public FigureList getRandomFigure() {
```

Comment:
```suggestion
    public Figure getRandomFigure() {
```
you should return a random Figure with random fields